kolibri/plugins/base.py

```diff
@@ -10,10 +10,10 @@
 logger = logging.getLogger(__name__)
 
 
-class MandatoryPluginMethodNotImplemented(Exception):
+class MandatoryPluginMethodNotImplemented(NotImplementedError):
 
     def __init__(self):
-        NotImplemented.__init__(self, "Plugin needs to define this method")
+        super(MandatoryPluginMethodNotImplemented, self).__init__("Plugin needs to define this method")
```

Comment:
Ah, no, let me try it out. I assumed it would work. I tested an error inheriting from NotImplementedError in the console, but I didn't override the __init__ function.